index.html

```diff
@@ -230,10 +252,11 @@ <h5 class="program__content--header">CC Ignite</h5>
                       title="program"
                       class="program__content--icon"
                     />
-                    <h5 class="program__content--header">CC Exhibition</h5>
+                    <h5 class="program__content--header">Interviews</h5>
                     <p class="program__content--text">
-                      Am I Ready to Date? What is the purpose of dating? Find
-                      answers to these questions here
+                      Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                      Nobis quae ratione accusantium optio, ipsum minus sit
+                      vitae soluta maxime officia?
```

Comment:
- [ ] Great job so far 👍 Kindly ensure that all your program cards have the same height as in the original design. This will improve the UI of your site 👍 
- Your design:
![Annotation 2022-05-16 190119](https://user-images.githubusercontent.com/85113567/168656309-4eb1a839-2237-400b-90bc-3d690a19120f.jpg)
- Required design:
![Annotation 2022-05-16 190104](https://user-images.githubusercontent.com/85113567/168656368-2e157d98-ebb4-4492-94a5-594d89df1936.jpg)